stock.html

```diff
@@ -0,0 +1,1096 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
This is your organisation specific html file (money mine was a SRMKZILLA event )

Reply:
delete this

Reply:
Noted.